src/test/java/com/rocketden/main/util/ProblemTestMethods.java

```diff
@@ -0,0 +1,63 @@
+package com.rocketden.main.util;
+
+import com.rocketden.main.dto.problem.CreateProblemRequest;
+import com.rocketden.main.dto.problem.ProblemDto;
+import com.rocketden.main.dto.problem.ProblemInputDto;
+import com.rocketden.main.model.problem.ProblemDifficulty;
+import com.rocketden.main.model.problem.ProblemIOType;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class ProblemTestMethods {
```

Comment:
Good idea on creating this - perhaps we can write methods for these basic setups for other mocked tests as well, as we talked about today!